config/initializers/flipper.rb

```diff
@@ -43,7 +43,7 @@
       # default feautures to enabled for development and test only
       Flipper.enable(feature) if Rails.env.development? || Rails.env.test?
     end
-  rescue ActiveRecord::NoDatabaseError, ActiveRecord::StatementInvalid
```

Comment:
catching the all of the db errors pre-migrations was a game of whack-a-mole

Reply:
Maybe add a `Raven.tags_context(feature: 'flipper')` so we can easily find all flipper related errors in sentry?

Reply:
All of the errors were when I tried to run a rake task like `rake db:migrate` or `rake db:setup` before the flipper before the flipper tables existed.  So, I don't expect any of the errors to be in prod.